src/cobra/test/test_core/test_core_reaction.py

```diff
@@ -605,16 +642,16 @@ def test_model_less_reaction(model):
             reaction.reduced_cost
 
 
-def test_knockout(model):
+def test_knockout(model: Model) -> None:
+    """Test reaction knockouts."""
     original_bounds = dict()
     for reaction in model.reactions:
         original_bounds[reaction.id] = (reaction.lower_bound, reaction.upper_bound)
         reaction.knock_out()
         assert reaction.lower_bound == 0
         assert reaction.upper_bound == 0
     for k, (lb, ub) in six.iteritems(original_bounds):
```

Comment:
We don't need `six` any longer.
```suggestion
    for k, (lb, ub) in original_bounds.items():
```

Reply:
I was thinking that as well, but considering we still have Python 2.7 support, I didn't remove it. Maybe another PR to remove complete usage of `six` across codebase?

Reply:
Removing six in one big swoop sounds good to me, too. We don't support 2.7 any longer so that is not a concern.

Reply:
Okay then will do this in a separate PR.